snippets/csharp/VS_Snippets_CLR_System/system.Console-REFORMAT/CS/reformat.cs

```diff
@@ -1,27 +1,33 @@
-﻿using System;
+using System;
```

Comment:
There's also a Visual Basic version of this example at snippets/visualbasic/VS_Snippets_CLR_System/system.Console-REFORMAT/VB/reformat.vb that should be modernized.